Dockerfile

```diff
@@ -0,0 +1,47 @@
+FROM alpine:3.10
+
+ARG VIPS_VERSION
+
+RUN apk add --update --no-cache --virtual .build-deps \
+            build-base \
+            pkgconfig \
+            glib-dev \
+            gobject-introspection-dev \
+            expat-dev \
+            tiff-dev \
+            libjpeg-turbo-dev \
+            libexif-dev giflib-dev \
+            librsvg-dev \
+            lcms2-dev \
+            libpng-dev \
+            orc-dev \
+            libwebp-dev \
+            libheif-dev 
+            
+RUN wget https://github.com/libvips/libvips/releases/download/v${VIPS_VERSION}/vips-${VIPS_VERSION}.tar.gz
+
+RUN mkdir vips
+
+RUN tar xvzf vips-${VIPS_VERSION}.tar.gz -C vips --strip-components 1
+
+WORKDIR /vips
+
+RUN ./configure --enable-debug=no --without-python
+
+RUN make
+
+RUN make install
+
+RUN ldconfig /etc/ld.so.conf.d
+
+WORKDIR /
+
+RUN rm -rf vips
+
+RUN apk del .build-deps
+
+RUN apk add --update --no-cache libgsf glib gobject-introspection expat tiff libjpeg-turbo libexif giflib librsvg lcms2 libpng orc libwebp libheif
```

Comment:
The `libheif` package depends on `x265`, which contains patent-encumbered software. Are we OK with this?

Reply:
I can remove as well... I don't know much about libraries's licenses... I just put what made sense to me.

Reply:
Yes, we probably have to remove libheif as well, unfortunately.

Reply:
removed